ui/src/app/workflows/components/workflow-details/workflow-details.scss

```diff
@@ -93,3 +93,18 @@
     }
 
 }
+.badge{
+    position: absolute;
```

Comment:
I'm a bit worried about all this customization - you've 12 lines of CSS - what are you trying to do?

Reply:
This CSS adds the red badge:
<img width="65" alt="image" src="https://user-images.githubusercontent.com/614838/77949787-638e4200-727c-11ea-823b-78d30f143508.png">

Reply:
nice